apps/berik-inntektsmelding-service/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/berikinntektsmeldingservice/BerikInntektsmeldingService.kt

```diff
@@ -37,10 +39,12 @@ private const val UKJENT_VIRKSOMHET = "Ukjent virksomhet"
 
 data class Steg0(
     val kontekstId: UUID,
-    val avsenderFnr: Fnr,
+    val avsenderFnr: Fnr?,
```

Comment:
Hvis innsending skal få en egen service, som jeg støtter, så kan vi godt ha en kommentar for å huske å reverte denne endringen.